deployments/report-server-env-testnet.nix

```diff
@@ -0,0 +1,20 @@
+{ globals, ... }: with (import ./../lib.nix);
```

Comment:
is this needed considering zendesk only works with mainnet?

Reply:
Good point -- it is unneeded. I have removed this and killed the obsolete nixops resources.